tests/integration_tests/test_sklearn.py

```diff
@@ -367,7 +370,9 @@ def test_callbacks() -> None:
         callbacks=callbacks,  # type: ignore
     )
 
-    optuna_search.fit(X, y)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=ConvergenceWarning)
+        optuna_search.fit(X, y)
```

Comment:
[Note] The optimization result of `SGDClassifier` is out-of-scope of this unit test, and the change sounds reasonable.

```console
python3.8/site-packages/sklearn/linear_model/_stochastic_gradient.py:705: ConvergenceWarning: Maximum number of iteration reached before convergence. Consider increasing max_iter to improve the fit.
    warnings.warn(
```